process-sandbox/src/ipc.rs

```diff
@@ -55,5 +55,5 @@ pub fn generate_random_name() -> String {
     let mut hasher = sha2::Sha256::new();
     hasher.update(format!("{:?}{}{}", time, pid, mono));
     let hash = hasher.finalize();
-    format!("{:?}", hash)[0..16].to_string()
+    format!("{:?}", hex::encode(hash.as_slice()))[1..32].to_string()
```

Comment:
Why does it start from 1 and length is 31??
Starts from 0 and having length 32 is more natual.
Any reason for it?

Reply:
Was a mistake. I fixed.